sdk/keyvault/Azure.Security.KeyVault.Secrets/src/SecretClient.cs

```diff
@@ -352,7 +352,7 @@ private async Task<PageResponse<T>> GetPageAsync<T>(Uri firstPageUri, string nex
                 firstPageUri = new Uri(nextLink);
             }
 
-            using (Request request = CreateRequest(HttpPipelineMethod.Get, firstPageUri, addAPIVersion: false))
```

Comment:
We no longer need to be clever with when we add the API version?

Reply:
We never set this parameter to anything but `false`